templates/myaccount/dashboard.php

```diff
@@ -35,8 +35,14 @@
 
 <p>
 	<?php
+	/* translators: 1: Orders URL 2: Address URL 3: Account URL. */
+	$dashboard_desc = __( 'From your account dashboard you can view your <a href="%1$s">recent orders</a>, manage your <a href="%2$s">billing address</a>, and <a href="%3$s">edit your password and account details</a>.', 'woocommerce' );
+	if ( wc_shipping_enabled() ) {
+		/* translators: 1: Orders URL 2: Addresses URL 3: Account URL. */
+		$dashboard_desc = __( 'From your account dashboard you can view your <a href="%1$s">recent orders</a>, manage your <a href="%2$s">shipping and billing addresses</a>, and <a href="%3$s">edit your password and account details</a>.', 'woocommerce' );
+	}
 	printf(
-		__( 'From your account dashboard you can view your <a href="%1$s">recent orders</a>, manage your <a href="%2$s">shipping and billing addresses</a>, and <a href="%3$s">edit your password and account details</a>.', 'woocommerce' ),
+		esc_html( $dashboard_desc ),
```

Comment:
Same here, links won't survive through `esc_html`.

Reply:
@peterfabian, I fixed this also.